pom.xml

```diff
@@ -44,31 +44,17 @@
                 <directory>src/main/resources</directory>
                 <filtering>true</filtering>
                 <includes>
-                    <include>paper-plugin.yml</include>
+                    <include>plugin.yml</include>
                 </includes>
             </resource>
         </resources>
         <plugins>
-            <plugin>
-                <groupId>io.github.git-commit-id</groupId>
-                <artifactId>git-commit-id-maven-plugin</artifactId>
-                <version>5.0.0</version>
-                <executions>
-                    <execution>
-                        <id>get-the-git-infos</id>
-                        <goals>
-                            <goal>revision</goal>
-                        </goals>
-                        <phase>validate</phase>
-                    </execution>
-                </executions>
-            </plugin>
-
```

Comment:
This plugin is used to fetch commit data to append it to the plugin version. Why do you remove this?

Reply:
I accidentally deleted it

Reply:
You can add again, editing the pull request